src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -168,12 +174,13 @@ describe("when pressing bid button", () => {
         <ConfirmBid {...initialProps} />
       </BiddingThemeProvider>
     )
-    component.root.instance.setState({ conditionsOfSaleChecked: true })
+    component.root.findByType(Checkbox).props.onPress()
     relay.commitMutation = jest.fn()
+    const placeBidButton = findPlaceBidButton(component)
 
-    component.root.findByType(Button).instance.props.onPress()
+    placeBidButton.props.onPress()
 
-    expect(component.root.findAllByType(Spinner).length).toEqual(1)
+    expect(placeBidButton.props.loading).toEqual(true)
```

Comment:
We probably do not have to check the presence of the `<Spinner>` component here and can rely on the `loading` prop instead.